README.md

```diff
@@ -1,16 +1,26 @@
-# Title
+# Intro to Hashes
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
++ Explain when you would want to use a hash to store data
```

Comment:
We could make this learning goal a bit stronger by being explicit about the reason/s.

Reply:
```suggestion
+ Explain that hashes are useful to store unordered data
```